zig-mode.el

```diff
@@ -518,7 +518,7 @@ This is written mainly to be used as `end-of-defun-function' for Zig."
     (zig-format-on-save-mode 1)))
 
 ;;;###autoload
-(add-to-list 'auto-mode-alist '("\\.zig\\'" . zig-mode))
+(add-to-list 'auto-mode-alist '("\\.zig\\|zon\\'" . zig-mode))
```

Comment:
```suggestion
(add-to-list 'auto-mode-alist '("\\.zig\\|\\.zon\\'" . zig-mode))
```

I think matching for an additional dot before the extension is better so we don't match filenames such as `abczon`.

Reply:
```suggestion
(add-to-list 'auto-mode-alist '("\\.\\(zig\\|zon\\)\\'" . zig-mode))
```

alternatively, this

Reply:
Good point 👍